kvmagent/kvmagent/plugins/tensorfusion/executor.py

```
@@ -0,0 +1,459 @@
+
+        sanitized_env = env.copy()
+        sanitized_env['TF_LICENSE'] = '<REDACTED>'
+        sanitized_env['TF_LICENSE_SIGN'] = '<REDACTED>'
+        logger.info('starting tensor-fusion-worker: cmd=%s, cuda_index=%d, gpu=%s, vm=%s, env=%s' %
+                     (' '.join(cmd), cuda_index, pci_address, request.vm_uuid, sanitized_env))
+
+        log_fd = None
+        try:
+            log_fd = open(log_file, 'a')
+            try:
+                proc = subprocess.Popen(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)